src/coffee/connect/rest.coffee

```diff
@@ -193,7 +193,7 @@ class Rest
     throw new Error 'Query limit doesn\'t seem to be 0. This function queries all results, are you sure you want to use this?' if query.limit and query.limit isnt '0'
 
     params = _.extend {}, query,
-      limit: 50 # limit used for batches
+      limit: 500 # limit used for batches
```

Comment:
Yes, I can, but why not to put it here and fetch by 50 instead? This method is by default not optimized, right? When you fetch all, you want to fetch it with as minimum requests as possible.